<?xml version="1.0"?>
<launch>

  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video2" />
    <param name="image_width" value="800" />
    <param name="image_height" value="600" />
    <param name="camera_name" value="usb_cam" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="framerate" value="24" />
    <param name="pixel_format" value="yuyv"/>
    <param name="io_method" value="mmap"/>
  </node>

  <node name="data_saver" pkg="baxter_unity" type="data_saver.py" args="--file_name $(arg file_name)" required="true" output="screen" />
  
</launch>
